libvast/src/taxonomies.cpp

```diff
@@ -55,13 +58,24 @@ caf::error convert(const data& d, concepts_type& out) {
           auto concept_ = caf::get_if<std::string>(&c);
           if (!concept_)
             return make_error(ec::convert_error, "concept is not a string:", c);
-          out.data[*name].concepts.push_back(*concept_);
+          dest.concepts.push_back(*concept_);
         }
       } else {
         return make_error(ec::convert_error,
                           "concepts is not a list:", cs->second);
       }
     }
+    auto desc = c->find("description");
+    if (desc != c->end()) {
+      if (auto description = caf::get_if<std::string>(&desc->second)) {
+        if (dest.description.empty())
+          dest.description = *description;
+        else if (dest.description != *description)
+          VAST_WARNING_ANON("conflicting concept descriptions for", *name,
+                            ": \"" + dest.description + "\" and \"" + *description +
+                            "\"");
```

Comment:
This warning message does not follow our usual log message style, e.g., "<noun> encountered conflicting descriptions...".

Also, there's still a space between `*name` and `:`.